### ISSUE TITLE
Countdown audio does not play in analysis mode 

### ISSUE THREAD

**Issue by RubyMineshaft** [online-go/online-go.com#1505]
**Describe the bug**
The countdown audio does not play when the game is in analysis mode, causing some users to time out. 

**Expected behavior**
Sounds should still play in analysis mode

**Comment by RubyMineshaft** [comment]
I took a quick look around in the game view code, but nothing immediately stood out to me.  I'll take a closer look when I have more time unless someone has an idea what could be causing it.

**Comment by shinuito** [comment]
There's some other funny instances, where for example the stone sound won't play when you navigate back a few moves into a variation and then play a new stone.

I wonder could it be related.

**Comment by mmartin** [comment]
I think it's because of these lines:
```typescript
// check if we need to update our audio
if (this.mode === 'play' && this.engine.phase === 'play') {
```
https://github.com/online-go/goban/blob/main/src/GobanCore.ts#L2909

Somewhere around there is also the reason why countdown does not work properly with submit-move.

**Comment by anoek** [comment]
Fixed in online-go/goban#63c2db6bddcdfa891e554a0e4ff793273e3b16f9
